app/default_overrides.go

```diff
@@ -154,3 +158,33 @@ func DefaultBlockParams() tmproto.BlockParams {
 		TimeIotaMs: 1, // 1ms
 	}
 }
+
+func DefaultConsensusConfig() *tmcfg.Config {
+	cfg := tmcfg.DefaultConfig()
+	// Set broadcast timeout to be 50 seconds in order to avoid timeouts for long block times
+	// TODO: make TimeoutBroadcastTx configurable per https://github.com/celestiaorg/celestia-app/issues/1034
+	cfg.RPC.TimeoutBroadcastTxCommit = 50 * time.Second
+	cfg.RPC.MaxBodyBytes = int64(8388608) // 8 MiB
+	cfg.Mempool.TTLNumBlocks = 10
+	cfg.Mempool.MaxTxBytes = appconsts.DefaultMaxBytes
```

Comment:
Note, I have modified this value from 2MB to the DefaultMaxBytes. In the future, it would be good if this automatically adjusted to MaxBytes in the consensus params. There's the problem that if we increase the max square size in the future that everyone's local configs will only allows txs of 2MB